x-pack/plugins/lens/public/editor_frame_service/embeddable/embeddable.tsx

```diff
@@ -63,6 +66,10 @@ interface LensBaseEmbeddableInput extends EmbeddableInput {
   renderMode?: RenderMode;
   style?: React.CSSProperties;
   className?: string;
+  onBrushEnd?: (data: LensBrushEvent['data']) => void;
```

Comment:
@streamich Is there an issue with extending the embeddable input interface like this? I'm unsure whether we should keep this kind of API out of the embeddable interface and only expose it for the react component.